build/makelib/golang.mk

```diff
@@ -156,6 +156,14 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint:
+	@$(GOHOST) get -u github.com/golang/lint/golint
```

Comment:
ok.. i havent used glide. I am more used to godep. I will look into this nevertheless